packages/mds-metrics-sheet/utils.ts

```diff
@@ -1,6 +1,9 @@
 import { UrlOptions } from 'request'
 import requestPromise, { RequestPromiseOptions } from 'request-promise'
 
+// 120 seconds in ms
+export const MAX_TIMEOUT_MS = 120000
```

Comment:
Would this make sense to be an environment variable with a default instead? `Number(process.env.MAX_TIMOUT_MS ?? '120000')` or something like that to be able to adjust without committing a code change?

Reply:
Yup, agreed. We're really only maintaining this package for LA, it's going to be deprecated at some point in the near-future (hopefully).